ext4/tar2ext4/tar2ext4.go

```diff
@@ -195,11 +222,404 @@ func Convert(r io.Reader, w io.ReadWriteSeeker, options ...Option) error {
 	}
 
 	if p.appendVhdFooter {
-		return ConvertToVhd(w)
+		return ConvertToVhd(w, p.alignVHDToMB)
 	}
 	return nil
 }
 
+func findNextUnusedLogicalBlock(bytePosition uint64) uint64 {
+	block := uint64(bytePosition / compactext4.BlockSizeLogical)
+	if bytePosition%compactext4.BlockSizeLogical == 0 {
+		return block
+	}
+
+	return block + 1
+}
+
+func ConvertMultiple(multipleReaders []io.Reader, w io.ReadWriteSeeker, partitionGUIDs []string, diskGUIDString string, options ...Option) error {
```

Comment:
We can pass the IDs as `guid.GUID`, rather than passing strings and needing to parse them here.

Reply:
Oops, looks like a relic of the prototyping process :P I'll update.